### PR TITLE
ui changes

### PR DESCRIPTION
**Pull request by ganeshmurali-dev**
1. Comments rich text editor Alignment fixed.
2. Comments scroll issue fixed.
3. Valid urL validation missing so added that one
4. WeightageList Inline style change to SCSS new folder
5. Applications Details UI Fully Changed

### CODE REVIEW

File: src/pages/applications/Applications.jsx [hunk 1]
```diff
@@ -133,240 +139,252 @@ const Applications = () => {
...
                   </div>
-                </div>
-                <div className="personal-detial-background">
-                  <div className="details-section">
-                    <p className="personal-detial-title">Street Address</p>
-                    <p className="personal-detial-name">{details.address}</p>
+                  <div className="personal-detial-background">
+                    <div className="details-section">
+                      <p className="personal-detial-title">Gender</p>
+                      <p className="personal-detial-name">Male</p>
```
Comment (avinashdckap) [inline src/pages/applications/Applications.jsx:165]: Why it's hardcoded as male here?

**Comment by avinashdckap** [comment]
@Ganeshdckap  what is the point of showing education details when there is no data?
Screenshot from 2024-02-14 09-33-46

**Comment by avinashdckap** [comment]
@Parasuramandckap  on reloading the task/assesment page, edit icon is automatically selected on first but it's showing assignee section
Screenshot from 2024-02-14 09-36-56

**Comment by avinashdckap** [comment]
I still dont see , edit icon hover inspite of saying it multiple times @Ganeshdckap . Have u still not checked with sugumar?

**Comment by avinashdckap** [comment]
While reloading task, i could able to add task/assement , prevent or disable add task button when it's loading

**Comment by avinashdckap** [comment]
Screenshot from 2024-02-14 09-43-51
find a way to fix this asap @Parasuramandckap

**Comment by avinashdckap** [comment]
Screenshot from 2024-02-14 09-45-18
is this as per design @Ganeshdckap

**Comment by avinashdckap** [comment]
Screenshot from 2024-02-14 09-46-43
why assignee section is highlighted with border bottom when that is only one

File: src/pages/applications/Applications.jsx [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
 
-import { useParams,useNavigate } from "react-router-dom";
+import { useParams, useNavigate } from "react-router-dom";
 
 import axios from "axios";
-import { Pagination, Popover, Tag, Skeleton, Tooltip, notification } from "antd";
+import {
+  Pagination,
+  Popover,
+  Tag,
```
Comment (avinashdckap) [inline src/pages/applications/Applications.jsx:9]: why indentation & spaces are shown in review?
I've already said dont format the things which you've not worked! 
i dont  need spaces or indentation or formatter changes in review again @Ganeshdckap  @Parasuramandckap

Comment (avinashdckap) [reply src/pages/applications/Applications.jsx:9]: Please remove all the spaces & indentation in this PR which has been added by mistake using formatter and give it to me only the changes you've done @Ganeshdckap

**Review by avinashdckap — CHANGES_REQUESTED** [review]

File: src/pages/applications/Applications.jsx (hunk 1 above)
Comment (ParasuramanA) [reply src/pages/applications/Applications.jsx:165]: Backend not given anna

**Comment by avinashdckap** [comment]
> @Ganeshdckap what is the point of showing education details when there is no data? Screenshot from 2024-02-14 09-33-46

All fields are mandatory

**Pull request merged**